src/components/ui/form/TextArea/style.scss

```diff
@@ -0,0 +1,6 @@
+textarea {
+    width: 100%;
+    border-radius: 0.5rem;
+    padding: 0.5rem;
+    background-color: #f9fafb;
```

Comment:
Can we use color variable here?

Reply:
I've used colour variable here. Kindly review it.